experiments/AtmosLES/stable_bl_model.jl

```diff
@@ -202,9 +208,8 @@ end
 function surface_temperature_variation(state, t)
     FT = eltype(state)
     ρ = state.ρ
-    q_tot = state.moisture.ρq_tot / ρ
     θ_liq_sfc = FT(265) - FT(1 / 4) * (t / 3600)
-    TS = PhaseEquil_ρθq(param_set, ρ, θ_liq_sfc, q_tot)
+    TS = PhaseDry_ρθ(param_set, ρ, θ_liq_sfc)
```

Comment:
is there a need here for:

if bl.moisture isa DryModel
      TS = PhaseEquil_ρθq(param_set, ρ, θ_liq_sfc, q_tot)
else
      TS = PhaseDry_ρθ(param_set, ρ, θ_liq_sfc)
end

Reply:
Good point. This is something I also thought about. The problem is that `bl` is out of the scope of this function. Do you think it is worth modifying the inputs to the function to add the BalanceLaw @charleskawczynski?

Reply:
I think it’s fine to add the balance law here. Should be changes in just this file.